prevent-broker-scale-down-if-contains-partition .md

```diff
@@ -0,0 +1,63 @@
+
+# Prevent scale down of the brokers if they contain partitions
```

Comment:
I would talk in general about replicas not partition. They could be the same if the topic has 1 partition with 1 replica but not when it has more replicas. We should not scale down in any case, if the broker is hosting a replica leader or a follower (to avoid under-replicated partition or less than min ISR).